File: src/app/model/event.ts
export class Event {
  eventId?: number;
  title: string = '';
  description?: string = '';
  eventDateTime: string = ''; // ISO 8601 format (e.g., "2025-05-01T10:30:00")
  sendToEmail?: string;
  personalEvent: boolean = false;
  isActive: boolean = false; 
  createdTs?: string;
  lastUpdatedTs?: string;
}

File: schedulr/src/main/java/com/example/schedulr/model/Event.java
package com.example.schedulr.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDateTime;

@Entity
@Data
public class Event {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String title;
    private String description;
    private LocalDateTime eventDateTime;
    private String sendToEmail;     // whom to send reminder
    private boolean selfReminder;   // if true, reminder to self
    private boolean isActive;       // Event active or canceled

    private String createdTs;
    private String lastUpdatedTs;

//
//    @ManyToOne
//    private UserDetail createdBy;
}
